custom/enikshay/integrations/nikshay/repeater_generator.py

```diff
@@ -219,6 +227,117 @@ def handle_exception(self, exception, repeat_record):
                                 "hiv_test_nikshay_registered", "hiv_test_nikshay_error")
 
 
+@RegisterGenerator(NikshayFollowupRepeater, 'case_json', 'JSON', is_default=True)
+class NikshayFollowupPayloadGenerator(BaseNikshayPayloadGenerator):
+    def get_payload(self, repeat_record, test_case):
+        self.test_case = test_case
+        occurence_case = get_occurrence_case_from_test(test_case.domain, test_case.get_id)
+        episode_case = get_open_episode_case_from_occurrence(test_case.domain, occurence_case.get_id)
+        person_case = get_person_case_from_occurrence(test_case.domain, occurence_case.get_id)
+
+        test_case_properties = test_case.dynamic_case_properties()
+        episode_case_properties = episode_case.dynamic_case_properties()
+
+        interval_id, lab_serial_number, result_grade, dmc_code = self._get_mandatory_fields(test_case_properties)
```

Comment:
i think i did that because i thought it was good to have all payload specific properties being set in a single place to look at them all at once. So its fine with base properties being set separately since they are not payload specific. Just for better view.